README.adoc

```diff
@@ -130,38 +130,38 @@ TIP: If you are using https://pages.github.com/[GitHub Pages], plan ahead by kee
 
 === Prerequisites
 
-First you must install and configure {uri-nodejs-download}[Node.js] on your machine.
+First you must install and configure {uri-nodejs-download}[Node] on your machine.
 
 === Install
 
-Using npm:
+We recommend to install the dependencies in a project directory, such as the directory where your AsciiDoc presentations are stored.
+If you don't have a `package.json` file in your project directory, you can create one to eliminate warnings during the installation using:
 
-Create a directory to place slides. Initialize the directory to store npm packages within that directory. 
+ $ echo {} > package.json # or `npm init` if you prefer
 
- $ echo {} > package.json               # eliminates warnings, use `npm init` if you prefer
- $ npm i --save asciidoctor-reveal.js
+You can now install the dependencies:
 
-=== Rendering the AsciiDoc into slides
+ $ npm i --save asciidoctor @asciidoctor/reveal.js
 
-Once the package is installed, you can convert AsciiDoc files using reveal.js converter.
-Here we are converting a file named `presentation.adoc` into a reveal.js presentation using Node.js:
+=== Convert AsciiDoc into slides
 
-.asciidoctor-revealjs.js
-[source, javascript]
-----
-// Load asciidoctor.js and asciidoctor-reveal.js
-var asciidoctor = require('asciidoctor.js')();
-var asciidoctorRevealjs = require('asciidoctor-reveal.js');
-asciidoctorRevealjs.register()
+Once the dependencies are installed, verify that the `asciidoctor` command is available by running:
 
-// Convert the document 'presentation.adoc' using the reveal.js converter
-var options = {safe: 'safe', backend: 'revealjs'};
-asciidoctor.convertFile('presentation.adoc', options); // <1>
+ $ $(npm bin)/asciidoctor --version
```

Comment:
I will add a comment for Windows users as `$(npm bin)` won't work.

As an alternative, it's possible to define a task in the package.json:

```json
{
"scripts": {
    "asciidoctor": "asciidoctor"
  }
}
```

```
$ npm run asciidoctor -- --version
```

Or to use a relative path:

```
$ ./node_modules/.bin/asciidoctor --version
```

Please note that `./` is a Unix notation, not sure about the syntax on Windows.